File: Mathematical/Java/Can_Make_Triangle.java
class Solution 
{ 
    int[] canMakeTriangle(int A[], int N) 
    { 
        // code here
        int ans[] = new int[N-2];
        
        for(int i=0; i<A.length-2; i++) {
            
            int a = A[i];
            int b = A[i+1];
            int c = A[i+2];
            
            if((a+b)>c && (a+c)>b && (b+c)>a) {
                ans[i] = 1;
            }
            else {
                ans[i] = 0;
            }
        }
        return ans;
    }
}

File: Mathematical/Python/Can_Make_Triangle.py
class Solution:
    def canMakeTriangle(self, A, N): 
        #code here
        ans = []
        
        for i in range(N-2):
            a = A[i]
            b = A[i+1]
            c = A[i+2]
            
            if ((a+b)>c and (a+c)>b and (b+c)>a):
                ans.append(1)
            else:
                ans.append(0)
        
        return ans
